src/universal/containers/Action/ActionContainer.js

```diff
@@ -27,6 +28,10 @@ export default class ActionContainer extends Component {
   };
 
   componentWillMount() {
+    const socket = socketCluster.connect();
+    socket.on('version', (version) => {
+      showWarning('You will be logged out and upgraded to version ' + version);
```

Comment:
you'll need some logic here to compare versions. if versions don't match, you'll want to trigger the logout from here so they don't get double toast.